.vscode/settings.json

```diff
@@ -28,5 +28,8 @@
     ["cx\\(([^)]*)\\)", "(?:'|\"|`)([^']*)(?:'|\"|`)"],
     ["cn\\(([^)]*)\\)", "(?:'|\"|`)([^']*)(?:'|\"|`)"],
     ["clsx\\(([^)]*)\\)", "(?:'|\"|`)([^']*)(?:'|\"|`)"]
-  ]
+  ],
+  "[typescript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  }
```

Comment:
Background not his change:

I use [`dprint`](https://dprint.dev) on my projects and so far VSCode has not being smart enough to automatically figure out when a project is using prettier or dprint. When I told it, it seems that it stored that information into the project local settings.

I assume this is harmless for everyone else on the team.

Reply:
Checked locally and it seems harmless. I assume most of us are using prettier by default anyway, so it doesn't really change anything ✅